GT4PY_VERSION.txt

```diff
@@ -1 +1 @@
-v17
+bfd178f35cdbb292067d215e1f64477d3028c029
```

Comment:
I assume you didn't intend to leave this here?

Reply:
so this was to test the new feature without having to build a gt4py release. But it needs a change there (to expose the API of normalized origins